tools/ci_build/github/azure-pipelines/linux-gpu-tensorrt-daily-perf-pipeline.yml

```diff
@@ -60,46 +46,11 @@ jobs:
   pool: 'Onnxruntime-Linux-GPU-TensorRT-Perf'
   
   variables:
-    - name: trtContainerVersion
-      ${{ if eq(parameters.RunNvidiaContainer, true)}}:
-        ${{ if eq(parameters.TrtVersion, '8.2.1.8') }}:
-          value: 21.12
-        ${{ elseif eq(parameters.TrtVersion, '8.0.1.6') }}:
-          value: 21.07
-      
-    - name: nvidia_dockerfile
-      ${{ if eq(parameters.RunNvidiaContainer, true)}}:
-        value: nvcr.io/nvidia/tensorrt:$(trtContainerVersion)-py3
-      
-
-    - name: dockerfile_path
-      ${{ if eq(parameters.RunNvidiaContainer, true) }}:
-        value: dockerfiles
-      ${{ else }}:
-        value: tools/ci_build/github/linux/docker
-
-    - name: dockerfile
-      ${{ if eq(parameters.RunNvidiaContainer, true) }}:
-        value: $(dockerfile_path)/Dockerfile.tensorrt
+    - name: branchName
```

Comment:
Note that we will no longer have to specify the branch name in the pipeline variables. The branch used to run the pipeline is the branch used for everything else.

Reply:
@jywu-msft pointed out that we should still allow overriding the branch name in case we would like to test an old branch with the latest perf dashboard scripts.